lib/diagnostic.rb

```diff
@@ -131,6 +154,10 @@ def kick
     @damage *= multiplier
   end
 
+  def self.get_possible_moves
+    "kick, move, punch"
+  end
```

Comment:
The way to make the method chaining work is for `punch`, `kick`, and `move` to return `self`.That way, `ComboAttack.new.punch.move('left').kick.damage` is equivalent to
```ruby
```